crane-cloud/frontend
# feat: redesign the database information section

# Description

- The goal of this PR is to make the database details section look more compact but concise in aspects of presentation of information
- Also Improved and made better the click-to-copy functionality with help of our very own custom tooltip component, you can now easily click to copy any value if you want to!

## Type of change


## Trello Ticket ID

https://trello.com/c/ub8RF7Ce

## How Can This Been Tested?

- Run this branch locally and check out the `databases settings` on any created database and click around the database information section, check items such as `database name`, `database user`, `host`, `port` and `database password`

## Checklist:


# Screenshots

<img width="1012" alt="image" src="https://github.com/crane-cloud/frontend/assets/63339234/96101621-d9cc-4ba7-ada9-56edbabcaadd">
